scripts/generators/protobuf.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import yaml
+
+from os.path import join, exists
+from generators import ecs_helpers
+
+
+field_type_to_protobuf_type = {
+    'binary': 'string',
+    'boolean': 'bool',
+    'byte': 'int32',
+    'data_nanos': 'long',
+    'date': 'string',
+    'double': 'double',
+    'float': 'float',
+    'geo_point': 'string',
+    'half_float': 'float',
+    'integer': 'int32',
+    'ip': 'string',
+    'keyword': 'string',
+    'long': 'int64',
+    'object': 'map <string, string>',
```

Comment:
Does this support multiple of object nestings?

Reply:
It doesn't. Currently (once i fixed a bug) there are only two fields in the generated protobuf definition that use `map <string, string>`: `labels`, and `container.labels`. I think the lack of nesting will work for those, but it could potentially be an issue.